src/services/pathfinding.service.ts

```diff
@@ -1,7 +1,19 @@
 import { getDistanceFromLine, getDistance } from "geolib";
 import PriorityQueue from "ts-priority-queue";
 import _ from "lodash";
-import { TravelEdge, Location, TravelNode, PQItem } from "../types/main";
+import {
+  TravelEdge,
+  Location,
+  TravelNode,
+  PQItem,
+  POI,
+  Line,
+  POICategory,
+  FloorPath,
+  ConnectorPOI
+} from "../types/main";
+import * as floors from "../constants/floors.data";
+import { POIInfo } from "../constants/poi.data";
```

Comment:
I believe these 2 can be merged as ` import { POIInfo, buildingFloors } from "../constants" `

Reply:
Is this because of the new index files?

Reply:
Yeah exactly, probably need to change `floors` to `buildingFloors` too